src/layout/Hero/HeroImage.svelte

```diff
@@ -14,10 +14,10 @@
 	}
 
 	.hero-img {
-		overflow: hidden;
+		overflow:hidden;
```

Comment:
Irrelevant change, incorrect formatting

Reply:
my bad I did this by mistake and while adding the commit, I didn't see it. I'm doing a new PR.

Reply:
Ohh I did that by mistake, my bad. I have implemented the changes now:)